templates/main.html

```diff
@@ -47,8 +47,32 @@ <h3>
             {{/leaders}}
           </ul>
         </div>
-      {{/orgs}}
+      {{/withLeader}}
     </div>
+    {{#noLeader.length}}
+      <h4>Organization with no listed leader</h4>
```

Comment:
Organization(s)

Also `<h2`> probably looks better, let's test it!

Reply:
doesn't it look to big?

Reply:
Yeah :/
Maybe try h3?

also "leaders"